README.md

```diff
@@ -1 +1,138 @@
 # web3-file
```

Comment:
Needs badges

Reply:
I will add them as follow up PR, after initial release to already have the npm one in place